paasta_tools/cli/cmds/mark_for_deployment.py

```diff
@@ -1064,15 +1061,15 @@ def on_exit_deploying(self) -> None:
         self.cancel_paasta_status_reminder()
 
     def on_enter_start_rollback(self) -> None:
-        # TODO: Pass image_version, update messages to use DeploymentVersion
         self.update_slack_status(
-            f"Rolling back ({self.deploy_group}) to {self.old_git_sha}"
+            f"Rolling back ({self.deploy_group}) to {self.old_deployment_version}"
         )
         self.mark_for_deployment_return_code = mark_for_deployment(
             git_url=self.git_url,
             deploy_group=self.deploy_group,
             service=self.service,
             commit=self.old_git_sha,
+            image_version=self.image_version,
```

Comment:
This should probably be old_image_version, right?  
I was surprised this wasn't caught by tests, and saw down below it's because we are validating the wrong values are passed mocked `mark-for-deployment`! added a comment.

Reply:
My bad, completely missed that! I've just made the change to fix it 👍